.github/workflows/auto-assign.yml

```
@@ -0,0 +1,19 @@
+    types: [opened]
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+    - name: 'Auto-assign issue'
+      uses: pozil/auto-assign-issue@v1
+      with:
+          repo-token: ${{ secrets.AUTO_ASSIGN }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)